packages/expo-blob/ios/Blob.swift

```diff
@@ -0,0 +1,107 @@
+import Foundation
+import ExpoModulesCore
+
+public class Blob: SharedObject {
```

Comment:
```suggestion
internal class Blob: SharedObject {
```

nit: personally i'm leaning toward having explicitly "internal". that could prevent symbols from polluting the whole app namespace.